src/main/java/ua/softserveinc/tc/util/EventBuilder.java

```diff
@@ -0,0 +1,85 @@
+package ua.softserveinc.tc.util;
+
+import ua.softserveinc.tc.constants.EventConstants;
+import ua.softserveinc.tc.entity.Event;
+import ua.softserveinc.tc.entity.Room;
+
+import java.util.Date;
+
+public class EventBuilder {
```

Comment:
It is not best implementation of builder pattern. There we are creating one needless object of EventBuilder.